storybookjs/storybook
# Mobile device view: toggling stories panel with ☰ button

Issue:

## What I did
* adapted Header component  to have ☰ button when on mobile device
* make stories-panel toggled by clicking ☰ button
* added helper to detect whether the storybook is open on mobile device (this is done through userAgent of window)
* adjusted styles 

Now on mobile devices the layout looks something like this when storybook is rendered:
![image](https://user-images.githubusercontent.com/3757759/38176267-ac776684-35f4-11e8-9bd0-2c486937f02e.png)

## How to test
Open storybook on mobile device

Is this testable with Jest or Chromatic screenshots?
yes (I added some new tests)
Does this need a new example in the kitchen sink apps?
no
Does this need an update to the documentation?
not sure
If your answer is yes to any of these, please make sure to include it in your PR.
